Source/MediaInfo/Multiple/File_Mpeg_Descriptors.cpp

```diff
@@ -2866,7 +2866,7 @@ void File_Mpeg_Descriptors::Descriptor_81()
     BS_Begin();
     Get_S1 (3, sample_rate_code,                                "sample_rate_code"); if (sample_rate_code<4) {Param_Info2(AC3_SamplingRate[sample_rate_code], " Hz");}
     Skip_S1(5,                                                  "bsid");
-    Get_S1 (6, bit_rate_code,                                   "bit_rate_code"); Param_Info2(AC3_BitRate[bit_rate_code]*1000, " Kbps");
+    Get_S1 (6, bit_rate_code,                                   "bit_rate_code"); if (bit_rate_code<19) {Param_Info2(AC3_BitRate[bit_rate_code]*1000, " Kbps");}

```

Comment:
Not important (no need to edit), but for next time, prefer
Param_Info2C(bit_rate_code<19, AC3_BitRate[bit_rate_code]*1000, " Kbps");
(the test code is deleted by preprocessor in that case, when small package is requested)